app/scripts/controllers/eventRegistrations.js

```diff
@@ -232,7 +232,7 @@ angular.module('confRegistrationWebApp')
           var registrantIndex = _.findIndex($scope.registrations[index].registrants, { 'id': registrantData.id });
           $scope.registrations[index].registrants[registrantIndex] = registrantData;
         }).error(function(){
-          modalMessage.error('Error: registrant data could be be retrieved.');
+          modalMessage.error('Error: registrant data could not be retrieved.');
```

Comment:
Thanks for fixing the typos.  Will this endpoint ever return a json error message?  If so, let's add that to this modal.